pkgs/profpatsch/nman/nman.1

```diff
@@ -0,0 +1,194 @@
+.Dd $Mdocdate$
+.Dt NMAN 1
+.Os
+.Sh NAME
+.Nm nman
+.Nd nix-shell for man pages
+.Sh SYNOPSIS
+.Nm
+.Op Fl h
+.Ar ATTR
+.Op Ar PAGE | SECTION Op PAGE
+.Sh DESCRIPTION
+.Nm
+is a simple tool allowing to open man pages of packages
+which are not installed using the power of the
+.Xr nix 1
+package manager.
+It allows you to specify a package and a man
+page and opens it directly using
+.Xr man 1 ,
+essentially implementing what you would expect
+.Ql nix-shell -p foo --run 'man foo'
+to do.
+.Pp
+You can invoke
+.Nm
+in three different ways:
```

Comment:
four different ways

I’d structure this section the other way around, from simpler to more complex. Since users will usually start with the obvious and only need the more complicated stuff if the obvious isn’t the case.